.vscode/extensions.json

```
@@ -18,5 +18,6 @@
     "mikestead.dotenv",
     "rebornix.ruby",
     "wingrunr21.vscode-ruby",
+    "firsttris.vscode-jest-runner"
```

pour info, c'est JSON with comment, qui autorise de terminer les listes avec une virgule, qui simplifie le diff lorsqu'on ajoute quelque chose à la liste

```suggestion
    "firsttris.vscode-jest-runner",
```